lib/decoder.js

```diff
@@ -213,6 +224,10 @@ Decoder.prototype.decode = function(len, data, callback) {
     }
   }
 
+  if(len === 11) {
```

Comment:
Its a assumption made for DPT1,DPT5 and DPT9. Somehow I think that we don't have any conflicts for these type.

I don't feel comfortable to merge this, because if other guys try to use other DPTs its kind of broken for them. I think its better to give the decision to the user for these types. I understand your argument that there are no other decode methods yet. I guess it would be only a matter of time we get the first issue about this.

I want to hear what past commiters think about this. Maybe we get some replies.
@stasee @snowdd1 @marcopiraccini @anlumo